mmedit/datasets/pipelines/augmentation.py

```diff
@@ -50,6 +50,12 @@ class Resize:
         interpolation (str): Algorithm used for interpolation:
             "nearest" | "bilinear" | "bicubic" | "area" | "lanczos".
             Default: "bilinear".
+        backend (str | None): The image resize backend type. Options are `cv2`,
+            `pillow`, `None`. If backend is None, the global imread_backend
+            specified by ``mmcv.use_backend()`` will be used.
+            Default: "cv2".
```

Comment:
Again, the default backend should be None. Otherwise you will change behavior of old codes

Reply:
OK

Reply:
In fact, if backend=None, cv2 will be used in mmcv.resize